lib/Zonemaster/Backend/DB.pm

```diff
@@ -260,9 +260,11 @@ sub get_test_params {
 
     my $result;
     eval {
-        # TODO: do we use "encode_utf8" as this was the case in PostgreSQL
-        #       (see commit diff)
-        $result = decode_json( $params_json );
+        if ( utf8::is_utf8( $params_json ) ) {
+            $result = decode_json( encode_utf8( $params_json ) );
+        } else {
+            $result = decode_json( $params_json );
+        }
```

Comment:
#571 also handles "$result->{results}". Is it certain that it is not needed to include that? Just to be safe?

Reply:
This part of the code has not been modified with the latest release, so I thought to keep it that way for the fix release and update it with v2022.1. However, as said in #934, I'm fine with replacing this PR (#932) with #934 and include it into v2021.2.1.

Reply:
That seems reasonable.